httpie/sessions.py

```diff
@@ -79,6 +79,13 @@ def update_headers(self, request_headers: RequestHeadersDict):
             if name == 'User-Agent' and value.startswith('HTTPie/'):
                 continue
 
+            if name == 'Cookie':
```

Comment:
There’s no guarantee on the capitalization (i.e., you probably wanna `.lower()` the name before comparing it).

Reply:
Actually, maybe there’s is some implied guaranteed since the it seems to work for the `User-Agent` check above :) But since header names are generally considered case-insensitive, it would be good to always normalize before comparing header names.